src/java.base/share/classes/java/util/Locale.java

```diff
@@ -70,131 +70,150 @@
- *   <code>[a-zA-Z]{2,8}</code>.  Note that this is not the full
- *   BCP47 language production, since it excludes extlang.  They are
- *   not needed since modern three-letter language codes replace
- *   them.</dd>
+ *   <dd> <em>Syntax:</em> Well-formed {@code language} values have the form {@code [a-zA-Z]{2,8}}.</dd>
+ *   <dd> <em> BCP 47 deviation:</em> this is not the full BCP 47 language production, since it excludes
+ *   <a href="https://datatracker.ietf.org/doc/html/rfc5646#section-2.2.2">extlang</a>
+ *   (as modern three-letter language codes are preferred).</dd>
  *
- *   <dd>Example: "en" (English), "ja" (Japanese), "kok" (Konkani)</dd>
+ *   <dd> <em>Example:</em> "en" (English), "ja" (Japanese), "kok" (Konkani)</dd>
  *
  *   <dt><a id="def_script"><b>script</b></a></dt>
  *
- *   <dd>ISO 15924 alpha-4 script code.  You can find a full list of
- *   valid script codes in the IANA Language Subtag Registry (search
- *   for "Type: script").  The script field is case insensitive, but
+ *   <dd> ISO 15924 alpha-4 script code.</dd>
+ *
+ *   <dd> <em>Case convention:</em> {@code script} is case insensitive, but
  *   {@code Locale} always canonicalizes to title case (the first
  *   letter is upper case and the rest of the letters are lower
  *   case).</dd>
  *
- *   <dd>Well-formed script values have the form
- *   <code>[a-zA-Z]{4}</code></dd>
+ *   <dd> <em>Syntax:</em> Well-formed {@code script} values have the form {@code
+ *   [a-zA-Z]{4}}</dd>
  *
- *   <dd>Example: "Latn" (Latin), "Cyrl" (Cyrillic)</dd>
+ *   <dd> <em>Example:</em> "Latn" (Latin), "Cyrl" (Cyrillic)</dd>
  *
  *   <dt><a id="def_region"><b>country (region)</b></a></dt>
  *
- *   <dd>ISO 3166 alpha-2 country code or UN M.49 numeric-3 area code.
- *   You can find a full list of valid country and region codes in the
- *   IANA Language Subtag Registry (search for "Type: region").  The
- *   country (region) field is case insensitive, but
+ *   <dd> ISO 3166 alpha-2 country code or UN M.49 numeric-3 area code.</dd>
+ *
+ *   <dd> <em>Case convention:</em> {@code country (region)} is case insensitive, but
  *   {@code Locale} always canonicalizes to upper case.</dd>
  *
- *   <dd>Well-formed country/region values have
- *   the form <code>[a-zA-Z]{2} | [0-9]{3}</code></dd>
+ *   <dd> <em>Syntax:</em> Well-formed {@code country (region)} values have the form {@code
+ *   [a-zA-Z]{2} | [0-9]{3}}</dd>
  *
- *   <dd>Example: "US" (United States), "FR" (France), "029"
+ *   <dd> <em>Example:</em> "US" (United States), "FR" (France), "029"
  *   (Caribbean)</dd>
  *
  *   <dt><a id="def_variant"><b>variant</b></a></dt>
  *
- *   <dd>Any arbitrary value used to indicate a variation of a
- *   {@code Locale}.  Where there are two or more variant values
- *   each indicating its own semantics, these values should be ordered
- *   by importance, with most important first, separated by
- *   underscore('_').  The variant field is case sensitive.</dd>
```

Comment:
This part "separated by underscore('_')" is missing in the revised doc

Reply:
I intentionally omitted this because

- I thought it was redundant as the syntax already follows a few lines below.
- The `Locale` syntax allows both `_` and  `-`, not just `_`.

However, if you think it should remain, I'll restore it as `separated by ('_'|'-')`.